app/src/main/res/layout/activity_create_lobby.xml

```diff
@@ -87,4 +88,28 @@
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/checkBoxIsPublic" />
 
+    <TextView
+        android:id="@+id/txtMaxRounds"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_marginTop="16dp"
+        android:text="@string/createLobbyMaxRoundsText"
+        android:textSize="20sp"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/sliderMaxPlayers" />
+
+    <com.google.android.material.slider.Slider
```

Comment:
The current default value seems to be 1. Can we change this to something like 5?